src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/CoinviewRule.cs

```diff
@@ -120,6 +108,12 @@ public override async Task RunAsync(RuleContext context)
             else this.Logger.LogTrace("BIP68, SigOp cost, and block reward validation skipped for block at height {0}.", index.Height);
         }
 
+        /// <summary>Checks if transaction if final.</summary>
+        protected virtual bool IsTxFinal(Transaction transaction, RuleContext context)
+        {
+            return transaction.IsFinal(context.ValidationContext.ChainedHeaderToValidate);
```

Comment:
What's currently in `transaction.IsFinal` the same as X?

Reply:
Yeah